recipes/presto-server/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "presto-server" %}
+{% set version = "0.284" %}
+
+package:
+  name: {{ name|lower }}
+  version: "{{ version }}"
+
+source:
+  # Build needs a proper Git checkout.
+  git_url: https://github.com/prestodb/presto
+  git_rev: "{{ version }}"
+
+
+build:
+  number: 0
+  noarch: python
```

Comment:
You have files outside the `site-packages`. Did you mean to make this `noarch: generic`?

Reply:
I couldn't find any docs on `generic`. The final application will be architecture independent because it uses Java and Python.

Reply:
I don't know where the docs for it are, but you really want `noarch: generic` here. It is the same as `noarch: python` but with files that exist outside of the `site-packages` folder.